File: android/app/src/main/java/com/lightningkite/kwifttemplate/vg/DrawableDemoVG.shared.kt
package com.lightningkite.kwifttemplate.vg

import android.view.View
import com.lightningkite.kwift.views.ViewDependency
import com.lightningkite.kwift.views.ViewGenerator
import com.lightningkite.kwifttemplate.layouts.DrawableDemoXml

class DrawableDemoVG() : ViewGenerator() {
    override val title: String get() = "Drawable Demo"

    override fun generate(dependency: ViewDependency): View {
        val xml = DrawableDemoXml()
        val view = xml.setup(dependency)

        return view
    }
}

File: ios/Kwift Template/com/lightningkite/kwifttemplate/vg/DrawableDemoVG.shared.swift
//Package: com.lightningkite.kwifttemplate.vg
//Converted using Kwift2

import Foundation
import Kwift
import KwiftMaps



public class DrawableDemoVG: ViewGenerator {
    
    
    override public var title: String {
        get {
            return "Drawable Demo"
        }
    }
    
    override public func generate(dependency: ViewDependency) -> View {
        var xml = DrawableDemoXml()
        var view = xml.setup(dependency)
        return view
    }
    override public func generate(_ dependency: ViewDependency) -> View {
        return generate(dependency: dependency)
    }
    
    override public init() {
        super.init()
    }
}
 
